docs/course_materials/exercises/01_Linux/check_class_exercise_1.py

```diff
@@ -0,0 +1,28 @@
+import os.path
```

Comment:
I believe you don't need this here anymore

Reply:
Done